trelabels.js

```diff
@@ -214,6 +214,6 @@ function setStyle(newStyle) {
   style = newStyle;
 }
 
-$(function () {
+$(window).on('load', function() {
```

Comment:
We just need the DOM to be loaded, can you give me more details about this change ?

Reply:
I had this error:

![Anmerkung 2019-05-09 105304](https://user-images.githubusercontent.com/2011264/57440962-435f2a00-7249-11e9-9b4c-df06c92ae8ad.jpg)

The function keepTrelabelsButton() was executed to soon, so my goal was to call it later when everything is really loaded.

Reply:
Executed after the DOM is loaded but before #header is a node, that's also weird...

Reply:
It's time for another guess ! In the new version of Trello, `#header` is added to the DOM via JS ?

Reply:
Ok, my bad, even in the old version, it's added via JS.

Reply:
The new version uses react, so I would say "yes" without taking a deeper look.

Reply:
A better way to handle this is to loop until `#header` appears, like what's done in [`addTrelabelsButton()`](https://github.com/fredericseiler/trelabels/blob/master/trelabels.js#L71-L74).

Reply:
What's your `window.trelloVersion` result in the Chrome Console ? build-3124 ?

Reply:
Yes, "build-3124". A loop would also work and might be more robust.

Reply:
How about just moving the `#header` loop from `addTrelabelsButton()` to `init` and call that instead? Like:

```
function init() {
	
  if ($('#header').length === 0) {
    setTimeout(init, 1000);
    return;
  }
	
  $('head').append(font);
  addTrelabelsButton();
  keepTrelabelsButton();
  listenForClick();
  listenForResize();
}
```

Reply:
Yep, makes more sense to delay the whole `init`.

So, we have the same Trello build but not the same content. Can you give it a try in incognito mode, without any other extension enabled ?

Reply:
I tried it five times in incognito and always got the react version.

Reply:
Side node : the "old" version is also using React.

Reply:
Ah ok, I didn't notice :)

I am not really familiar with Github... how can I update the pull request with the changes we discussed? Just a commit in my fork to the master branch?

Reply:
Yes, the PR will auto-update when you push to the same branch (master) in your fork.

You can also [allow me](https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to do the changes on your PR, if you prefer.